src/include/network/services/dns.rb

```diff
@@ -645,10 +645,6 @@ def storeHostnameGlobal(_key, event)
       nil
     end
 
-    def ReallyAbortInst
-      Popup.ConfirmAbort(:incomplete)
```

Comment:
It is used here https://github.com/yast/yast-firstboot/blob/master/src/lib/y2firstboot/clients/hostname.rb#L97

Reply:
so it is time to move it there, isn't it?